makefiles/host/rootfs.mk

```
@@ -56,3 +59,17 @@ build/nctrl/target/release/nctrl: build/nctrl/.copy_stamp
 	FUSE_CROSS_STATIC_PATH=./thirdparty/ \
 	FUSE_CROSS_STATIC_LIB=fuse \
 	cargo build --release --target=armv7-unknown-linux-musleabihf
+
+$(OPENOCD_SOURCE): build/root.fs/.base_install
+	@mkdir -p $(@D)
+	rm -rf $@
+	git clone https://repo.or.cz/openocd.git $@
+	(cd $@ && git reset --hard $(OPENOCD_VERSION))
+	touch $@/.scmversion
+
+$(OPENOCD_SOURCE)/.build_stamp: $(OPENOCD_SOURCE) build/root.fs/.base_install
+	(cd $(@D) && ./bootstrap)
+	(cd $(@D) && ./configure --target=$(CROSS) --prefix=$${PWD}/../root.fs/ --enable-sysfsgpio )
```

Sorry, `host` not target, also we probably want to install under `usr`.
```suggestion
	(cd $(@D) && ./configure --host=$(CROSS) --prefix=$${PWD}/../root.fs/usr/ --enable-sysfsgpio )
```

---

`$(CROSS)` doesn't work because of the `-` suffix.